Jenkinsfile

```diff
@@ -1,4 +1,6 @@
 
 buildNPM {
   publishModDescriptor = 'yes'
-}  
+  runLint = 'yes'
```

Comment:
This didn't appear to have any effect - there must be some config in Jenkins that isn't part of the repo.